problemsets-mat223/modules/module4-exercises.tex

```diff
@@ -1,17 +1,114 @@
 \begin{exercises}
+	% Topics:
+	% dot products, orthogonality
+	% unit vectors, vector length,
+	% angle between vectors, distance between vectors
+	% lines, planes, hyperplanes in normal form
 	\begin{problist}
-		\prob  Example problem with parts
+		\prob  Compute the following dot products.
 		\begin{enumerate}
-			\item   The first part
-			\item   The second part
+			\item   $\mat{9\\4} \cdot \mat{10\\-3}$
+			\item   $\mat{1\\36\\2} \cdot \mat{0\\0\\1}$
+			\item   $\mat{7\\6\\-3} \cdot \left(\mat{5\\11\\-1} + \mat{-2\\-6\\-1}\right)$
+			\item   $\mat{1\\3\\0\\-5\\5} \cdot \mat{1\\2\\2\\1\\2}$
+			\item   $\frac{1}{2}\mat{2\\5\\4} \cdot \mat{1\\0\\-1}$
 		\end{enumerate}
-		\begin{solution}
-		Solutions to the parts
+		\prob Compute the length of the following vectors.
+		\begin{enumerate}
+			\item $\mat{2\\0}$
+			\item $\mat{1\\2\\3}$
+			\item $4\mat{5\\-6\\15\\2}$
+			\item $\mat{-\frac{\sqrt{10}}{10}\\-\frac{9\sqrt{10}}{50}\\\frac{12\sqrt{10}}{50}}$
+			\item $\mat{\frac{1}{3}\\\frac{\sqrt{3}}{3}\\0\\-\frac{1}{3}\\\frac{2}{3}}$
+		\end{enumerate}
+		\prob Find the cosine of the angle between the following vectors.
+		\begin{enumerate}
+			\item $\mat{1\\0}$ and $\mat{-3\\4}$
+			\item $\mat{1\\0\\1}$ and $\mat{-5\\4\\-3}$
+			\item $\mat{1\\2\\3}$ and $\mat{-1\\-1\\2}$
+			\item $\mat{2\\4\\6}$ and $\mat{\frac{3}{2}\\3\\\frac{9}{2}}$
+			\item $\mat{0\\1\\0\\1}$ and $\mat{-1\\1\\3\\0}$
+		\end{enumerate}
+		\prob For each vector, find two \emph{unit} vectors orthogonal to it
+		\begin{enumerate}
+			\item $\mat{0\\1}$
+			\item $\mat{1\\2}$
+			\item $\mat{1\\3\\5}$
+			\item $\mat{-13\\-4\\5}$
+			\item $\mat{0\\1\\1\\\frac{1}{2}}$
+		\end{enumerate}
+
+		\prob Compute the distance between the following vectors
+		\begin{enumerate}
+			\item $\mat{-1\\1}$ and $\mat{-1\\-4}$
+			\item $\mat{2\\-6\\5}$ and $\mat{-4\\7\\-3}$
+			\item $\mat{1\\1\\1}$ and $\mat{-1\\-1\\-1}$
+			\item $\mat{6\\0\\2\\1}$ and $\mat{0\\1\\3\\-1}$
+			\item $\mat{0\\0\\0\\0\\0}$ and $\mat{1\\1\\1\\1\\1}$
+		\end{enumerate}
+
+		\prob
+		\begin{enumerate}
+			\item Which vector out of $\mat{1\\0}$, $\mat{0\\1}$, and $\mat{4\\1}$
+			has a direction closest to $\mat{3\\5}$?
+			\item Which vector out of $\mat{2\\3\\4}$, $\mat{1\\-1\\-1}$, and $\mat{-3\\0\\1}$
+			has a direction closest to $\mat{1\\0\\1}$?
+		\end{enumerate}
+
+		\prob
```

Comment:
Let's change this to "For each plane specified, express the plane in both vector form and normal form." Then you can just specify the planes and you can combine it with the next question.